b2/sync.py

```diff
@@ -775,17 +780,19 @@ def make_folder_sync_actions(source_folder, dest_folder, args, now_millis, repor
     ]:
         raise NotImplementedError("Sync support only local-to-b2 and b2-to-local")
     for (source_file, dest_file) in zip_folders(source_folder, dest_folder):
-        if source_folder.folder_type() == 'local':
-            if source_file is not None:
-                reporter.update_compare(1)
-        else:
-            if dest_file is not None:
-                reporter.update_compare(1)
-        for action in make_file_sync_actions(
-            sync_type, source_file, dest_file, source_folder, dest_folder, args, now_millis
-        ):
-            yield action
+        if not any([is_excluded(ex, source_file) or
```

Comment:
shouldn't only the source side be excluded? Also, please use a generator expression rather than a list comprehension here - there is no need to allocate a temporary list.